src/NUnitFramework/framework/Constraints/EqualConstraint.cs

```diff
@@ -74,10 +74,18 @@ public EqualConstraint(object? expected)
         /// Gets a value indicating whether to compare ignoring white space.
         /// </summary>
         /// <value>
-        ///   <see langword="true"/> if comparing ignoreing white space; otherwise, <see langword="false"/>.
+        ///   <see langword="true"/> if comparing ignoring white space; otherwise, <see langword="false"/>.
         /// </value>
         public bool IgnoringWhiteSpace => _comparer.IgnoreWhiteSpace;
 
+        /// <summary>
+        /// Gets a value indicating whether to compare after normalizing newlines.
```

Comment:
The code isn't "normalizing newlines" or line endings, but "ignoring different line endings"
Normalize indicates it changes them before comparing which is something not done deliberately.
Maybe rename all "NormalizeLineEndings" into "IgnoreDifferentStyleLineEndings".
It also keeps the naming of the properties consistent where the modifiers ignores an aspect, being it case, or line endings or white space.

Reply:
We have `IgnoreCase`, not `IgnoreDifferentCases`, so I went with `IgnoreLineEndingFormat`.